free-1.md

```diff
@@ -10,34 +10,48 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This DSL would help users learn more about flavor pairings and make suggestions if given specific foods. The users would be able to figure out foods or beverages that would pair well with a given item. For example, if a person were hosting a dinner party and didn't know what to serve with the main dish, they could receive suggestions about flavors or dishes that would be good accompaniments. 
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Flavor pairing is a unique domain and a DSL would help to educate users about successful pairings. It is appropriate for users because they would be able to interact with pairings in various ways (learning about more pairings, getting pairing suggestions, etc).
```

Comment:
It's definitely a very unique and limited DSL. I know personally I had a few ideas with a very small domain, and I felt perhaps a language wasn't exactly the best choice halfway through responding to these questions. I had a hard time imaging a DSL being built for something that perhaps a web or mobile application could do so much easier. I'm not even sure whether it would still be considered a DSL or not if you made it more like an application.